java/managed-vms/README.md

```diff
@@ -114,6 +166,7 @@ This app is a [Jetty](http://www.eclipse.org/jetty/) based [Servlet](http://www.
   * POST - Will convert the JSON to ColumnFamily : Qualifier and write the data
   * DELETE - Will remove the row.
 
+1. The URL should be either localhost:8080, docker:8080, or  https://<projectID>.appspot.com
 1. `curl -H "Content-Type: application/json" -X POST -d '{"username":"red","id":535}' http://localhost:8080/json/blueword`
```

Comment:
Does this work as shown, or do you also need to include a column family in each of the column qualifiers (for example, `cf1:username` rather than `username`)? If this works as shown, which column family is used by default?

I assume the only column families that you can use are `cf1`, `cf2`, `cf3`, and `cf4`, since those are the ones you specify when you create the table. True? If so, that'd be good to mention.